
Token indices sequence length is longer than the specified maximum sequence length for this model (1267 > 1024). Running this sequence through the model will result in indexing errors
--- Logging error ---
Traceback (most recent call last):
  File "/var/conda/lib/python3.10/logging/__init__.py", line 1104, in emit
    self.flush()
  File "/var/conda/lib/python3.10/logging/__init__.py", line 1084, in flush
    self.stream.flush()
OSError: [Errno 5] Input/output error
Call stack:
  File "/var/conda/lib/python3.10/threading.py", line 973, in _bootstrap
    self._bootstrap_inner()
  File "/var/conda/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/var/conda/lib/python3.10/site-packages/ipykernel/parentpoller.py", line 38, in run
    get_logger().warning("Parent appears to have exited, shutting down.")
Message: 'Parent appears to have exited, shutting down.'
